### DISCUSSION TITLE
Made with Budibase Footer in 2.10.10

### DISCUSSION THREAD

**Discussion by tglynx** [Budibase/budibase#11817] [General]
Hi, 

i noticed a new footer in 2.10.10 that is really annoying when using budibase on a mobile phone. I have lots of forms in my app which have been optimized for this specific purpose and the new footer screws it all. 
I understand that marketing is necessary, however, is it really necessary to have this footer on every screen?

How can i get rid of the footer? 

If the answer is a paid license: 
We really like Budibase and use it in our nonprofit organisation with > 100 Users. However, we can not afford 16$ per User just to remove the branding. I really hope this is just a campaign that is removed in a future version!

In addition to this i did put lots of efford in creating mobile apps for Android and iOS which embedd the budibase screens. 
The new footer just kills the ability to use this as marketing links to the website are not allowed like this.

Thanks for help or feedback!
Michael

**Comment by ConorWebb96** [comment]
Hey @tglynx,

The only way to remove the footer is with a paid-tier of Budibase. The footer isn't a campaign as such; we've decided that if users are getting value from Budibase community (e.g. by having an app in production), it would be a win-win if knowledge that it's Budibase under-the-hood was shared (as we get no revenue from community) - we feel a footer is the best way to do that.

The hope isn't to make it too intrusive nor disrupt any of your current apps negatively; so I'm keen to see if there's a way to make it read better on mobile forms, specifically. Can we either hop on a call or you share an app export?

We do offer non-profit pricing - would you like me to set up a call with our sales team?

Best,
Conor

**Reply by tglynx** [reply]
Hi @ConorWebb96 , 

really appreciate your answer, thanks! I do understand that building this environment is so much efford, and that there is no revenue from community members that only use the free self hosted version. However, i try to give back as much as possible to the community and the product itself by helping others i  implementing things in budibase here on github because i really liked how budibase was working, and i think this is how open source works.
It was one key point in my decision when choosing budibase for our project that there is no branding in the views itself!

I read about the non-profit pricing and i will definitely have a call with your sales team 👍 
I will evaluate what budget we have to get rid of the footer - if any (it really kills lots of space)!

However, i just wanted to express my feelings about the new footer in this post!

Have a nice day,
Michael

**Reply by renrom** [reply]
I mentiond this also a couple of days agoo. It seems it is visible since the 2.10.10. version. As i do not mind the marketing purpose of this footer, but i do mind the height of the footer. For only the text 'made with budibase' it has a lot of white space above and benieth it. I would at least suggest make the footer a bit smaller?

**Comment by JCV67** [comment]
It would be great if in automation sent email we   had this footer instead of the current header.

**Comment by gmentzik** [comment]
Just noticed this footer after updating to 2.11. This footer is taking too much screen real estate and does not give good impression, especially if you have taken lots of time to design a completely custom layout. Please try to find a more discreet approach to advertise. End users usually do not care about how things are implemented as long as it works and is pleasing. But the ones working on dev environment care and will promote a good app!

**Reply by spitoutzi** [reply]
Use embed to remove this footer (name of css element may be another):
```
<style>
.free-footer.svelte-far5dn {
display:none;
}
</style>
```

**Reply by ConorWebb96** [reply]
Hey all, 

I just wanted to say that this is against Budibase's ToS. You can read up on the ToS in its entirety [here](https://budibase.com/self-host-master-terms/) -  Section 4.2

**Reply by spitoutzi** [reply]
Thanks @ConorWebb96 , I really didn't know about it. Sorry 😔

**Reply by gmentzik** [reply]
Could you please point to appropriate paragraph of ToS?

**Reply by mjashanks** [reply]
Hey @gmentzik - Paragraph 4.2 details this in the link above (I changed the link).

However, I am clarifying this further below, as different terms apply in different scenarios. 

### Budibase Cloud
Whether free or paid, users are under the following terms: https://budibase.com/terms-of-service/

These terms do not currently include a restriction for circumventing the branding - but we will add that soon.

### Self-hosted

If you are on a free plan - i.e. open-source, your only terms are those listed in the repo license (mostly-GPLv3): https://github.com/Budibase/budibase/blob/master/LICENSE 

If you are on a paid plan, your terms are: https://budibase.com/self-host-master-terms/
